### PR TITLE
Improve CHANGELOGs

### PR DESCRIPTION
**Pull request by hcsa73**
- Add core improvement
- Add change type

### CODE REVIEW

File: services/mongodbflex/CHANGELOG.md [hunk 1]
```diff
@@ -4,8 +4,8 @@
 
 ## v0.10.0 (2024-01-05)
 
-- Added method `ListMetrics` (which accepts the desired metric: `cpu`, `database`, `disk-iops`, `disk-use`, or `exec-time`) as an argument and replaces `ListCPUMetrics`, `ListDatabaseStorageMetrics`, `ListDiskIOPSMetrics`, `ListDiskUsageMetrics`, and `ListExecutionTimesMetrics`, respectivelly
-- Removes the previously deprecated method `GetStorage`, replaced by `ListStorages`
+- **Feature:** Added method `ListMetrics` (which accepts the desired metric: `cpu`, `database`, `disk-iops`, `disk-use`, or `exec-time`) as an argument and replaces `ListCPUMetrics`, `ListDatabaseStorageMetrics`, `ListDiskIOPSMetrics`, `ListDiskUsageMetrics`, and `ListExecutionTimesMetrics`, respectivelly
```
Comment (joaopalet) [inline services/mongodbflex/CHANGELOG.md:7]: This is also a breaking change, as it removes the `ListCPUMetrics`, `ListDatabaseStorageMetrics`, `ListDiskIOPSMetrics`, `ListDiskUsageMetrics`, and `ListExecutionTimesMetrics` methods

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,41 +1,42 @@
 ## Release (2024-01-09)
 
 ### Highlights
-
+- `core`: [v0.7.5](core/CHANGELOG.md#v0.72024-01-09)
```
Comment (joaopalet) [inline CHANGELOG.md:4]: Shouldn't this be
```suggestion
- `core`: [v0.7.5](core/CHANGELOG.md#v075-2024-01-09)
```
?

File: services/mongodbflex/CHANGELOG.md (hunk 1 above)
Comment (hcsa73) [reply services/mongodbflex/CHANGELOG.md:7]: Done in the main changelog as well

File: CHANGELOG.md (hunk 2 above)
Comment (hcsa73) [reply CHANGELOG.md:4]: Yes it should, not sure what happened here...

**Review by joaopalet — APPROVED** [review]

**Pull request merged**